tests/unit/EntityInfo.spec.ts

```diff
@@ -77,8 +78,7 @@ describe('Entity Info component in a Correction as a named company', () => {
 
   it('renders the business name and numbers', () => {
     expect(wrapper.find('#entity-legal-name').text()).toBe('My Mock Name Inc.')
-    // this is actually business id but should be tax id (see ticket 15122)
-    // expect(wrapper.find('#entity-business-number').text()).toBe('1234567')
+    expect(wrapper.find('#entity-business-number').text()).toBe('123456789')
```

Comment:
This comment went into "Pending" status... not sure why that happened. I musta clicked the wrong button. DoH!

Reply:
It probably went into Pending if you clicked "Start a review" instead of "Add single comment".

I usually add single comments so they're separate from my review.

Reply:
> If this isn't an entity-business-number any more, shouldn't the label be updated to reflect it's a taxId? (similar question for line 152, below)

What I meant is that the displayed value was actually the business id. But it should be the business number.

It's not labelled Business Number consistently everywhere (I think).

Reply:
Oh, okay. It appeared to me that it was testing an entity-business-number to a taxId.

Reply:
Business Number and Tax ID are the same thing. For display purposes, we call it Business Number. In the business object/schema, it's called Tax ID.

It's easily confused with Business ID, which is different.